CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+This document describes how to go about contributing to the ISOBlue-Avena repository as well as defining how we manage pull requests, releases, and other collaborative features
+
+
+## General Guidelines
+If you are looking to help us, please visit our [issue tracker](https://github.com/OATS-Group/isoblue-avena/issues). If you would like to suggest or discuss a feature/bug, please email our mailing list found here https://groups.google.com/forum/?oldui=1#!forum/isoblue. Please ensure you mention the 'Avena' version of ISOBlue.
+
+
+## Release Branches and Versioning
+The master branch will track the current unstable version of the codebase (equivalent to the `dev` branch in many other repositories). Stable versions will be maintained using branches in the form of `release/xx.yy.zz`, similar to the [Gitlab Flow](https://docs.gitlab.com/ee/topics/gitlab_flow.html) style. This is primarily for the ability to cherry-pick bug fixes into previous releases. 
+
+Releases will be numbered in the [Semantic Versioning](https://semver.org/) style. Version `XX.YY.ZZ` is major version `XX`, minor version `YY`, and bugfix `ZZ`. Major versions incorporate breaking changes, minor versions introduce new features, and bugfixes... fix bugs
+
+
+## Feature Branches and Pull Requests
+Members of the ISOBlue-Avena repository will use feature/bugfix branches in the form of `feature/feature-name` or `bugfix/bug-name`. Outside contributors forking the repository are encouraged to do the same
+
+
+Once a feature/bugfix branch as reached maturity and has been properly [tested](#Testing) a pull request will be created requesting to merge the branch into master. Members of the ISOBlue-Avena repository will review the code and changes may be be requested. Given all requested changes are addressed and all tests pass, a member of the repository will approve the changes and the branch will be merged into master
+
+
+## Testing
+As of writing, testing is currently very fluid. All outside contributors are expected to test their contributions properly before opening a pull request. Once a pull request is opened, our github actions (once completed) will preform supplemental testing. These tests must pass before a pull request will be considered. The master branch (depending on the scope of the changes, occasionally feature branches as well) will be tested on farm equipment within the Purdue OATS Research group. Once the software preforms sufficiently well in the field, a [release branch](#Release-Branches-and-Versioning) will be created and that version of the codebase will be considered 'stable'
+
+
+## Code of Conduct
```

Comment:
Many other `contributing.md`s that I saw had substantial 'Code of Conduct' sections detailing things like discrimination, inclusion, trolling, inappropriate behavior, etc. I think linking to Purdue's policy on all that that should be good enough, plus it is probably better written than I could ever do. Is the new S20 RCR the best thing to link?

Reply:
I agree. Wording is pretty clear and is _I think_ what we are looking for a Code of Conduct.

Reply:
:+1: